### PR TITLE
[#155858296] new recipe for reporting endpoint

### PR DESCRIPTION
**Pull request by jamiethezim**

### CODE REVIEW

File: recipes/recipe-18-report-request.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Example:
+
+```
+curl -H "Authorization: Bearer ${TOKEN}" \
+-X POST \
+-d startDate=2017-07-01 \
+-d endDate=2017-07-31 \
+-d fields=FIRST_NAME \
+-d fields=LAST_NAME \
```
Comment (brandonsov) [inline recipes/recipe-18-report-request.md]: I assume the second fields would overwrite the first fields key. Could you delete one of these?

File: recipes/recipe-18-report-request.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+https://services.sheerid.com/rest/0.5/report
+```
+
+There are a number of parameters that can be used to customize the report output. Personally Identifiable Information (PII) can be included on a per field basis or all available data can be requested by using the fields=”*” wildcard. The PII returned will depend on the SheerID user roles the report requester has. Metadata submitted with requests can be included on the report by adding the metadata keys in the report request as a comma delimited list
```
Comment (brandonsov) [inline recipes/recipe-18-report-request.md]: Could you add what to pass to the endpoint to allow PII?

File: recipes/recipe-18-report-request.md (hunk 1 above)
Comment (jamiethezim) [reply recipes/recipe-18-report-request.md]: fields can be listed by comma delimited list

File: recipes/recipe-18-report-request.md (hunk 2 above)
Comment (jamiethezim) [reply recipes/recipe-18-report-request.md]: no parameter is required, the request is likely going to be made by a non-staff user, which doesn't need a reason to view PII.

**Pull request merged**